### DISCUSSION TITLE
Is it possible to Get the output of a report via powershell ?

### DISCUSSION THREAD

**Discussion by dionrowney** [thycotic-ps/thycotic.secretserver#405] [Q&A : Module] [answered]
Is there a documented way to run a report and retrieve the report output... maybe as a csv ? via the power shell cmdlets?

**Comment by dionrowney** [comment, marked as answer]
Nevermind - I got it 
Export-TssReport LOL

**Comment by jagger** [comment]
This is already supported via `Invoke-TssReport`, which executes a report and returns the rows as PSCustomObject. Example: `Invoke-TssReport -TssSession $session -ReportName GroupMembershipReportByGroup -Parameters @{ Group = 1 }` — pipe to `Export-Csv` if you want a CSV file. Docs: https://thycotic-ps.github.io/thycotic.secretserver/commands/reports/Invoke-TssReport. Filing a docs follow-up to surface this more prominently from the Reports section. Please reopen if `Invoke-TssReport` does not cover your case.
